app/components/SampleTrees/SampleTreesCount.js

```diff
@@ -42,14 +49,47 @@ const TreeNumberSelection = ({ sampleTreesCount, selectedTreeCount, setSelectedT
             </TouchableOpacity>
           );
         })}
+      <View
+        style={[
+          styles.treeCountInputSelection,
+          isCustomSelected ? styles.treeCountSelectionActive : {},
+        ]}>
+        <TextInput
+          style={{
+            borderBottomColor: isCustomSelected ? 'white' : Colors.TEXT_COLOR,
+            borderBottomWidth: 1,
+            paddingVertical: 0,
+            color: 'white',
+            alignSelf: 'center',
+            // marginBottom: 4,
+          }}
+          selectionColor={'white'}
+          keyboardType={'numeric'}
+          onFocus={() => setIsCustomSelected(true)}
+          textAlign={'center'}
+          onChangeText={(text) => {
```

Comment:
Not sure if there already is some validation of the input, but it should be limited to numbers >0 and maybe some upper limit?

Reply:
Yeah. I will add the validation too.